scripts/mover.py

```diff
@@ -24,16 +32,22 @@
     sys.exit(1)
 
 
-def filter_torrents(torrent_list, timeoffset_from, timeoffset_to):
+def filter_torrents(torrent_list, timeoffset_from, timeoffset_to, cache_mount):
     result = []
     for torrent in torrent_list:
         if torrent.added_on >= timeoffset_to and torrent.added_on <= timeoffset_from:
-            result.append(torrent)
+            if not cache_mount or exists_in_cache(cache_mount, torrent.content_path):
```

Comment:
`not cache_mount` is the same, but also checks for empty string, is there something I'm missing?

Reply:
if someone pass empty string it will try checking if exists in cache and that will return false. So the torrent will be skipped, which I would like to avoid and better stop it. But I do not mind your suggestion if you think it is better